src/main/kotlin/frc/robot/RobotContainer.kt

```diff
@@ -118,8 +123,7 @@ object RobotContainer {
             .onTrue(extender.reset(operatorController.povUp().negate()))
     }
 
-    fun getAutonomousCommand(): Command = autoChooser.get()
-
+    fun getAutonomousCommand(): () -> Command = { autoChooser.get() }
```

Comment:
1. I believe you can do:
```suggestion
    fun getAutonomousCommand(): () -> Command = autoChooser::get
```
2. Why is that needed? What was the problem with it previously?